customer.py

```diff
@@ -0,0 +1,86 @@
+import requests
+from datetime import datetime
+
+class Customer:
+    def __init__(self, url="http://localhost:5000", selected_customer=None):
+        self.url = url
+        self.selected_customer = selected_customer
+
+    def create_customer(self, name, postal_code, phone, \
+                    registered_at=None, videos_checked_out_count=None):
+        query_params = {
+            "name": name,
+            "postal_code": postal_code,
+            "phone": phone,
+            "registered_at": registered_at,
+            "videos_checked_out_count": videos_checked_out_count
+        }
+        response = requests.post(self.url+"/customers", json=query_params)
+        return response.json()
+
+    def list_customers(self):
+        response = requests.get(self.url+"/customers")
+        return response.json()
+
+    def get_customer(self, name, id):
```

Comment:
In `main.py` you call this function with either an id or name, but not both.  So it's crashing.